Documentation/spelling_wordlist.txt

```diff
@@ -1,4 +1,3 @@
-Admin
```

Comment:
Did you intentionally remove this one?

Reply:
Nope, I made this mistake during rebase. Thanks for pointing out :+1: